contract/contracts/contracts.sol

```diff
@@ -0,0 +1,49 @@
+pragma solidity ^0.6.0;
+
+interface TokenFactoryInterface {
+    // Get created token count
+    function totalTokenCount() external returns(uint256);
+
+    // Get token address by index
+    function tokenOf(uint256 id) external returns(address);
+
+    // get token amount of creator
+    function tokenAmountOf(address creator) external returns(uint256);
+
+    // get a creator token of specific id
+    function creatorTokenOf(address creator, uint256 id) external returns(address);
```

Comment:
Creator can create multiple token, so when creator access a management page it will be listed and select single address. 
So this seems necessary.

Reply:
Ok, I understand.
In that case, wouldn't you need a function similar to `totalTokenCount` like `totalCreatorTokenCount`? To get number of tokens that creator created.

Reply:
> number of tokens 

This means total supply? IMO its not necessary if it is.
Because when we get token address from this function, then we can fetch a call to actual token contract address and call `totalSupply`.

Reply:
No.
It was the same as this comment.
https://github.com/TART-tokyo/fan-stock/pull/1#discussion_r501034791

Reply:
Ah I understand.
`tokenAmountOf(creator)` perform that functionality.